packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -1,196 +1,212 @@
 import { css } from '@microsoft/fast-element';
-import { display } from '../utils/index.js';
+import { SystemColors } from '@microsoft/fast-web-utilities';
```

Comment:
Just a note that this package is going to be deprecated, nothing to do here but we will want a migration as an additional piece of work.

Reply:
Considering that these are already built-in constants, I'll update the styles to just use those regular CSS values.